plugins/database/mongodb/util.go

```diff
@@ -17,6 +17,16 @@ type mongoDBStatement struct {
 	Roles mongodbRoles `json:"roles"`
 }
 
+// writeConcern is the mgo.Safe struct with JSON tags
+// More info: https://godoc.org/gopkg.in/mgo.v2#Safe
+type writeConcern struct {
```

Comment:
Do we need this struct or can we just use the `mgo.Safe` version?

Reply:
The `mgo.Safe` struct doesn't have json tags for the fields :(

Reply:
Actually, we might not need the JSON tags if there's no snake-casing on the fields so this could probably go away.

Reply:
Okay, in that case i'd say it's fine to leave this struct as is

Reply:
We agreed offline that this is okay to remove since we are following mongoDB's naming convention (e.g. `wtimeout` is not snake-cased). In this case, we can use `mgo.Safe`'s struct directly.